src/client/sandbox/code-instrumentation/methods.js

```diff
@@ -1,12 +1,44 @@
 import SandboxBase from '../base';
-import { isNullOrUndefined, inaccessibleTypeToStr, isWindow, isDocument } from '../../utils/types';
+import { isNullOrUndefined, inaccessibleTypeToStr } from '../../utils/types';
 import { DOCUMENT_WRITE_BEGIN_PARAM, DOCUMENT_WRITE_END_PARAM, CALL_METHOD_METH_NAME } from '../../../processing/js';
+import { FOCUS_PSEUDO_CLASS_ATTR } from '../../../const';
+import { isWindow, isDocument, isDomElement } from '../../utils/dom';
+import { isIE } from '../../utils/browser';
 
 export default class MethodCallInstrumentation extends SandboxBase {
     constructor (sandbox) {
         super(sandbox);
 
         this.methodWrappers = {
+            // NOTE: When a selector that contains the ':focus' pseudo-class is used in the querySelector and
+            // querySelectorAll functions, the latter return an empty result if the browser is not focused.
+            // This replaces ':focus' with a custom CSS class to return the current active element in that case.
+            querySelector: {
+                condition: element => !isIE && (isDocument(element) || isDomElement(element)),
```

Comment:
In the hammerhead we are using 'el' reduction:

 condition: el => !isIE && (isDocument(el) || isDomElement(el)),